src/components/homePage/DealsOfTheDay.js

```diff
@@ -18,6 +18,7 @@ const DealsOfTheDay = () => {
               <img src={i.image} alt={i.brand} />
               <p>{i.name}</p>
               <p>₹{i.mrp - (i.mrp * i.discount) / 100}</p>
+              <p>{i.discount}% off</p>
```

Comment:
What does this `i` stand for? Couldnt it be a more meaningful variable name?

Reply:
Yes sure